### ISSUE TITLE
JSONDecodeError in version 0.20

### ISSUE THREAD

**Issue by ghost** [semgrep/semgrep#1567]
Receiving an error running Semgrep. Below is the first lines of the install followed by the run with the error information output.

Install:
Run pip3 install semgrep
Collecting semgrep
  Downloading semgrep-0.20.0-cp36.cp37.cp38.py36.py37.py38-none-manylinux1_x86_64.whl (29.6 MB)
  ...

Run:

```sh
$ semgrep --config=r2c --strict --sarif ./opnsesame > ./SemgrepCodeAnalysis.sarif
using config from https://semgrep.dev/p/r2c. Visit https://semgrep.dev/registry to see all public rules.
downloading config...
running 78 rules...
Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.8.5/x64/bin/semgrep", line 8, in <module>
    sys.exit(main())
  File "/opt/hostedtoolcache/Python/3.8.5/x64/lib/python3.8/site-packages/semgrep/__main__.py", line 17, in main
    cli()
  File "/opt/hostedtoolcache/Python/3.8.5/x64/lib/python3.8/site-packages/semgrep/cli.py", line 354, in cli
    semgrep.semgrep_main.main(
  File "/opt/hostedtoolcache/Python/3.8.5/x64/lib/python3.8/site-packages/semgrep/semgrep_main.py", line 356, in main
    rule_matches_by_rule, debug_steps_by_rule, semgrep_errors = CoreRunner(
  File "/opt/hostedtoolcache/Python/3.8.5/x64/lib/python3.8/site-packages/semgrep/core_runner.py", line 489, in invoke_semgrep
    findings_by_rule, debug_steps_by_rule, errors = self._run_rules(
  File "/opt/hostedtoolcache/Python/3.8.5/x64/lib/python3.8/site-packages/semgrep/core_runner.py", line 459, in _run_rules
    rule_matches, debugging_steps, errors = self._run_rule(
  File "/opt/hostedtoolcache/Python/3.8.5/x64/lib/python3.8/site-packages/semgrep/core_runner.py", line 365, in _run_rule
    output_json = self._run_core_command(
  File "/opt/hostedtoolcache/Python/3.8.5/x64/lib/python3.8/site-packages/semgrep/core_runner.py", line 299, in _run_core_command
```

**Comment by nbrahms** [comment]
This was fixed in https://github.com/returntocorp/semgrep/pull/1552, and will be available in 0.21

**Comment by minusworld** [comment]
0.21 will release on Tuesday! in the meantime, you can try `pip install semgrep==0.19.1`.
